kong/plugins/statsd/handler.lua

```diff
@@ -12,62 +12,118 @@ local string_gsub = string.gsub
 local pairs = pairs
 local NGX_ERR = ngx.ERR
 
+local function allow_user_metric(message, identifier)
+  if message.consumer ~= nil and
+          message.consumer[identifier] ~= nil then
+    return true, message.consumer[identifier]
```

Comment:
Its just for readability, I have been told before to keep it explicit  to make it more readable.